lib/pliny/metrics/backends/librato.rb

```diff
@@ -0,0 +1,50 @@
+begin
+  require "librato/metrics"
+  require "concurrent"
+rescue LoadError => error
+  puts <<-MSG
+! #{error.message}
+! Please add:
+!   gem "librato-metrics"
+!   gem "concurrent-ruby"
+! to your gemfile and run `bundle install` if you wish to use the Librato
+! metrics backend
+
+  MSG
+
+  raise
+end
+
+module Pliny
+  module Metrics
+    module Backends
+      class Librato
```

Comment:
Makes sense that `Backends::Logger` is a class as well. That aligns with `ErrorReporters::Rollbar`.